server/dataSimulator.js

```diff
@@ -31,6 +47,17 @@ var userGeneration = function() {
             }
           // for the csv 
           // usersArr.push([count, handle, name, 'PST', pubBool]);
+            var objIndex = {
+                index:  { _index: 'myindex', _type: 'mytype', _id: count }        
+            };
```

Comment:
You might want to rename the 'index' and 'type' to something more specific - especially if you're going to create multiple indices in Elasticsearch.